### PR TITLE
Feat/wagmi no modal example email passwordless

### PR DESCRIPTION
**Pull request by rtomas**
+ Example on how to use Wagmi + Web3auth + email Passwordless with web3auth no modal PnP SDK

### CODE REVIEW

File: web-no-modal-sdk/wagmi/wagmi-no-modal-example-email/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+[Join our Community Portal](https://community.web3auth.io/) to get support and stay up to date with the latest news and updates.
+
+This example demonstrates how to use Web3Auth with Wagmi Connector.
+
+## How to Use
+
+### Download Manually
+
+```bash
+npx degit Web3Auth/web3auth-pnp-examples/wagmi/wagmi-no-modal-example w3a-example
```
Comment (shahbaz17) [inline web-no-modal-sdk/wagmi/wagmi-no-modal-example-email/README.md:15]: The path here is wrong

Comment (shahbaz17) [reply web-no-modal-sdk/wagmi/wagmi-no-modal-example-email/README.md:15]: please update

File: web-no-modal-sdk/wagmi/wagmi-no-modal-example-email/src/Web3AuthConnectorInstance.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    }
+  });
+  web3AuthInstance.addPlugin(walletServicesPlugin);
+
+  return Web3AuthConnector({
+      web3AuthInstance,
+      loginParams: {
+        loginProvider: 'email_passwordless',
+        extraLoginOptions: {
+          login_hint: email // Email to send the OTP to
```
Comment (shahbaz17) [inline web-no-modal-sdk/wagmi/wagmi-no-modal-example-email/src/Web3AuthConnectorInstance.tsx:62]: Can we specify the code / link flow here too?

**Review by shahbaz17 — CHANGES_REQUESTED** [review]
@rtomas 

Please address the comments

File: web-no-modal-sdk/wagmi/wagmi-no-modal-example-email/src/Web3AuthConnectorInstance.tsx (hunk 2 above)
Comment (rtomas) [reply web-no-modal-sdk/wagmi/wagmi-no-modal-example-email/src/Web3AuthConnectorInstance.tsx:62]: additionalsParams is not available inside toruslabs/openlogin-utils/ExtraLoginOptions

**Comment by yashovardhan** [comment]
These examples have been updated with hooks. No need to these changes anymore.

**Pull request closed without merging**